app/models/concerns/event_associator.rb

```diff
@@ -0,0 +1,7 @@
+module EventAssociator
+  def assign_event(payload, event)
```

Comment:
What if we add this method to `GithubService`? That way it's inherited by the different services and we don't have to add a new concern here.

Reply:
Ok!